models/token.go

```diff
@@ -47,10 +48,26 @@ func (t *AccessToken) AfterSet(colName string, _ xorm.Cell) {
 	}
 }
 
+func isAccessTokenNameExist(uid int64, name string) (bool, error) {
+	tokens := make([]*AccessToken, 0, 5)
```

Comment:
I think we can make this function as simple as this (`.Get()` method):

https://github.com/gogs/gogs/blob/99f7b6c1f37e8c6e78009c5793652e402df6b9a4/models/user.go#L483

Which in this case, since it only has one line, we probably don't need this function to exist anymore.